projects/media-viewer/src/lib/annotations/annotation-set/annotation-create/highlight-create/highlight-create.directive.ts

```diff
@@ -22,7 +22,7 @@ export class HighlightCreateDirective implements OnInit, OnDestroy {
   rotate: number;
   allPages: object;
 
-  $subscription: Subscription;
+  private $subscriptions: Subscription[] = [];
```

Comment:
Weird that the $ sign is before rather than after (like in our applications) but its consistent across media-viewer so guess it was just the design choice of the previous team